devito/types/dense.py

```diff
@@ -881,27 +885,27 @@ class Function(DiscreteFunction, Differentiable):
     First-order derivatives through centered finite-difference approximations
 
     >>> f.dx
-    -f(x, y)/h_x + f(x + h_x, y)/h_x
+    Derivative(f(x, y), x)
     >>> f.dy
-    -f(x, y)/h_y + f(x, y + h_y)/h_y
+    Derivative(f(x, y), y)
     >>> g.dx
-    -0.5*g(x - h_x, y)/h_x + 0.5*g(x + h_x, y)/h_x
+    Derivative(g(x, y), x)
     >>> (f + g).dx
-    -(f(x, y) + g(x, y))/h_x + (f(x + h_x, y) + g(x + h_x, y))/h_x
+    Derivative(f(x, y) + g(x, y), x)
 
     First-order derivatives through left/right finite-difference approximations
 
     >>> f.dxl
-    f(x, y)/h_x - f(x - h_x, y)/h_x
+    Derivative(f(x, y), x)
```

Comment:
Since the output of `f.dxl` and `f.dxr` now look the same if would be nice to add something here illustrating the difference between them.

Reply:
I thought about it but I would rather not over clutter `Derivative` with a bunc of extra things like side/stagger/order/.... 
THis is the default `sympy.Derivative` repr and I think it makes sense

Reply:
Oh I agree with that, I was just wondering is the docstring should also include i.e. `f.dxl.stencil`.

Reply:
if we don't change the repr, then maybe we should just drop it...

Reply:
It's at least worth adding a comment along the lines of

```
Note that the fact that it's a left-derivative isn't captured in the representation. However, upon derivative expansion, this becomes clear

>>> f.dxl.evaluate
....
```

Reply:
Done